Packages/com.llealloo.audiolink/Editor/Scripts/AudioLinkShaderCompatabilityUtility.cs

```diff
@@ -121,5 +123,85 @@ private static void ReplaceInFile(string path)
                 File.WriteAllLines(path, shaderSource);
             }
         }
+
+        [InitializeOnLoadMethod]
+        private static void InitializeOnLoad()
+        {
+            EditorApplication.delayCall += ConvertShaderFiles;
+        }
+
+        private static string SetLightModeTag(string text, string from, string to)
+        {
+            return new Regex($"\"LightMode\"\\s*=\\s*\"{from}\"").Replace(text, $"\"LightMode\"=\"{to}\"");
+        }
+
+        [MenuItem("Tools/AudioLink/Convert Amplify Shaders Pipeline")]
+        public static void ConvertShaderFiles()
+        {
+            ShaderInfo[] shaders = ShaderUtil.GetAllShaderInfo();
```

Comment:
We might want to pop up an infobox when running this that informs the user that we are about to make changes to files on disk. It's a bit intrusive. Packages are supposed to be immutable so this is a bit strange, tbh, but I understand that there isn't really another way.